src/index.tsx

```diff
@@ -1,12 +1,17 @@
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
+import {
+  LoadingProvider,
+} from './components/LoadingContext/LoadingContext';
 
-import './styles/index.css';
-import './styles/todo-list.css';
-import './styles/filters.css';
+import 'bulma/css/bulma.css';
```

Comment:
Do you use it? And probably better to import minified file `import 'bulma/css/bulma.min.css';`
https://couds.github.io/react-bulma-components/?path=/story/welcome--page